sql/postgres/inspect.go

```diff
@@ -387,18 +387,17 @@ func (i *inspect) addIndexes(s *schema.Schema, rows *sql.Rows) error {
 				NullsLast:  nullslast.Bool,
 			})
 		}
-		switch {
-		case sqlx.ValidString(expr):
-			part.X = &schema.RawExpr{
-				X: expr.String,
-			}
-		case sqlx.ValidString(column):
```

Comment:
Why this change is needed?

Reply:
Each row of the indexesQuery returns an index part/column in a multicolumn index. As per the suggested query each part has a valid expression but column only if it is created with a column name:

```sql
LEFT JOIN pg_attribute a ON (a.attrelid, a.attnum) = (idx.indrelid, idx.key)
```

Reply:
Thanks for the explanation. If you don't mind, please use revert to the switch statement and I'll merge it.